internal/compiler/typeregister.rs

```diff
@@ -98,18 +98,32 @@ pub const RESERVED_ROTATION_PROPERTIES: &[(&str, Type)] = &[
     ("rotation-origin-y", Type::LogicalLength),
 ];
 
-pub const RESERVED_ACCESSIBILITY_PROPERTIES: &[(&str, Type)] = &[
-    //("accessible-role", ...)
-    ("accessible-checkable", Type::Bool),
-    ("accessible-checked", Type::Bool),
-    ("accessible-delegate-focus", Type::Int32),
-    ("accessible-description", Type::String),
-    ("accessible-label", Type::String),
-    ("accessible-value", Type::String),
-    ("accessible-value-maximum", Type::Float32),
-    ("accessible-value-minimum", Type::Float32),
-    ("accessible-value-step", Type::Float32),
-];
+pub fn reserved_accessibility_properties() -> impl Iterator<Item = (&'static str, Type)> {
+    [
+        //("accessible-role", ...)
+        ("accessible-checkable", Type::Bool),
+        ("accessible-checked", Type::Bool),
+        ("accessible-delegate-focus", Type::Int32),
+        ("accessible-description", Type::String),
+        ("accessible-label", Type::String),
+        ("accessible-value", Type::String),
+        ("accessible-value-maximum", Type::Float32),
+        ("accessible-value-minimum", Type::Float32),
+        ("accessible-value-step", Type::Float32),
+        ("accessible-action-default", Type::Callback { return_type: None, args: vec![] }),
+        ("accessible-action-increment", Type::Callback { return_type: None, args: vec![] }),
+        ("accessible-action-decrement", Type::Callback { return_type: None, args: vec![] }),
+        (
+            "accessible-action-set-value",
+            Type::Callback { return_type: None, args: vec![Type::Float32] },
```

Comment:
I wonder if the action should take a float or a string here.
It seems that the value on a line edit would be a string.
also the value property is a string.